package.json

```diff
@@ -143,6 +143,7 @@
     "start:punchout": "env-cmd --no-override -e punchout,$SPA_ENV nx serve storefrontapp --configuration=development",
     "start:prod": "env-cmd --no-override -e dev,b2c,$SPA_ENV nx serve storefrontapp --configuration=production",
     "start:pwa": " cd ./dist/storefrontapp/browser && http-server -p 4200 --silent --proxy http://localhost:4200?",
+    "start:new-auth-flow": "env-cmd --no-override -e s1,newAuthFlow,$SPA_ENV nx serve storefrontapp --configuration=development",
```

Comment:
There are several references to `newAuthFlow` throughout the PR.  They should be removed, correct?

Reply:
yeah unfortunately, after reverting changes and merging, it came back😅